common/service/config/ringpop/ringpop.go

```diff
@@ -156,41 +150,14 @@ func (factory *RingpopFactory) getRingpop() (*membership.RingPop, error) {
 
 func (factory *RingpopFactory) broadcastAddressResolver() (string, error) {
 	// This initial piece is copied from ringpop-go/ringpop.go/channelAddressResolver
-	var ch *tcg.Channel
-	var err error
-	if ch, err = factory.getChannel(factory.dispatcher); err != nil {
-		return "", err
-	}
-
-	return membership.BuildBroadcastHostPort(ch.PeerInfo(), factory.config.BroadcastAddress)
+	return membership.BuildBroadcastHostPort(factory.channel.PeerInfo(), factory.config.BroadcastAddress)
```

Comment:
@shawnhathaway, I am not sure if the comment above still relevant here.

Reply:
Yea comment isn't relevant anymore, feel free to pull out.